app/src/main/java/io/github/danthe1st/jdoc4droid/util/JavaDocDownloader.java

```diff
@@ -72,13 +71,6 @@
     private static final String METADATA_FILE_NAME = ".metadata";
     private static final Pattern JDK_JAVADOC_MAJOR_VERSION_PATTERN = Pattern.compile("[^0-9]*([0-9]+)([^0-9].*)?");
 
-    private InputStreamLoader inputStreamSupplier;
```

Comment:
Please don't move variables down below runnable blocks, not even static blocks.
Aside from that, my other statements about ordering apply here, too.